src/frontend/lib/resetToYourTeam.ts

```diff
@@ -0,0 +1,13 @@
+import {CommonActions, NavigationProp} from '@react-navigation/native';
+import type {AppStackParamsList} from '../sharedTypes/navigation';
+
+export function resetToYourTeam(
+  navigation: NavigationProp<AppStackParamsList>,
+) {
+  navigation.dispatch(
+    CommonActions.reset({
+      index: 2,
+      routes: [{name: 'Home'}, {name: 'ProjectSettings'}, {name: 'YourTeam'}],
+    }),
+  );
+}
```

Comment:
i dont thing we should pass the entire navigation object, since were only using the dipatch. You can do this for the typing:

```ts
type Navigation = NavigationProp<AppStackParamsList>;
type Dispatch = Navigation['dispatch'];

export function resetToYourTeam(dispatch: Dispatch) {
  dispatch(
    CommonActions.reset({
      index: 2,
      routes: [{name: 'Home'}, {name: 'ProjectSettings'}, {name: 'YourTeam'}],
    }),
  );
}
```